src/index.js

```diff
@@ -470,6 +470,9 @@ function createConfig(options, entry, format, writeMeta) {
 									preset: 'default',
 								}),
 						].filter(Boolean),
+						modules:
+							!!options['css-modules'] &&
+							(!!options.watch || { generateScopedName: '_[hash:base64:5]' }),
```

Comment:
Think we could use `--css-modules` here as the scoped name if its a String? (`--css-modules "_[hash:base64:5]"`)?

```suggestion
							(!!options.watch || {
								generateScopedName: typeof options['css-modules']=='string' ? options['css-modules'] : '_[hash:base64:5]'
							}),
```